selftests/functional/test_loader.py

```diff
@@ -215,6 +215,9 @@ def test_load_not_a_test(self):
     def test_load_not_a_test_not_exec(self):
         self._test('notatest.py', NOT_A_TEST, 'NOT_A_TEST')
 
+    @unittest.skipIf(os.environ.get("AVOCADO_CHECK_FULL") != "1",
+                     "Skipping test that take a long time to run, are "
+                     "resource intensive or time sensitve")
```

Comment:
@ldoktor the reference posted in the commit message shows that, even without a bug, this may not "finish immediattely".  We have followed the same approach on other time/resource sensitive tests.